apps/frontend-pwa/src/pages/course/[courseId]/index.tsx

```diff
@@ -269,9 +270,7 @@ function CourseOverview({ courseId }: Props) {
               {/* // TODO: update the translation strings as well, once this hard-coded content has been updated with a flexible implementation */}
               {course?.awards && course?.awards?.length != 0 && (
                 <div className="px-4 py-3 mt-4 bg-orange-100 border border-orange-200 rounded shadow md:mt-6">
-                  <H3 className={{ root: 'mb-2 text-base' }}>
-                    BF-Champion Awards
-                  </H3>
+                  <H3 className={{ root: 'mb-2 text-base' }}>Awards</H3>
```

Comment:
We should probably also translate the "Awards" string to also have a consistent German page?